packages/dotcom-ui-footer/src/components/partials.tsx

```diff
@@ -63,31 +63,27 @@ const FooterContents = ({ footerData }: TFooterContentsProps) => (
         // The Next navigation API splits footer links data into "columns"
         // <https://github.com/Financial-Times/next-navigation-api/blob/HEAD/server/transforms/footer.js>
         const submenu = item.submenu.items as TNavMenuItem[][]
-
         return (
-          <div
-            key={`group-${index}`}
-            className={`o-footer__matrix-group o-footer__matrix-group--${submenu.length}`}>
+          <div key={`group-${index}`} className={`o-footer__matrix-group o-footer__matrix-group--1`}>
```

Comment:
The className seems to have changed here, is this intentional?
It has gone from `o-footer__matrix-group o-footer__matrix-group--${submenu.length}` to `o-footer__matrix-group o-footer__matrix-group--1`